### PR TITLE
Added session

### PR DESCRIPTION
**Pull request by dhwanilvyas**

### CODE REVIEW

File: admin/index.php [hunk 1]
```diff
@@ -182,6 +188,7 @@
 			</section>
 			<br />
 			<a href="/" class="btn"> LOGOUT </a>
+			<?php session_destroy() ?>
```
Comment (PXgamer) [inline admin/index.php]: This results in being logged out when selecting an option in the admin page.

Would be better to add this as part of a new file (e.g. `/logout/index.php` that would have the following:

```php
<?php
session_destroy();
header('Location: /');
```

Or something similar, then change the logout button to link to `/logout/`.

**Review by owenvoke — APPROVED** [review]

File: admin/index.php (hunk 1 above)
Comment (owenvoke) [inline admin/index.php]: This results in being logged out when selecting an option in the admin page.

Would be better to add this as part of a new file (e.g. `/logout/index.php` that would have the following:

``` php
<?php
session_destroy();
header('Location: /');
```

Or something similar, then change the logout button to link to `/logout/`.

**Review by owenvoke — CHANGES_REQUESTED** [review]

File: admin/index.php (hunk 1 above)
Comment (dhwanilvyas) [reply admin/index.php]: I have updated my PR.

**Review by owenvoke — COMMENTED** [review]
Seems fine, I'll test it in a bit. :smile:

**Review by owenvoke — APPROVED** [review]

**Pull request merged**